types/is-function/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for is-function 1.0
+// Project: https://github.com/grncdr/js-is-function
+// Definitions by: Evangelos Zotos <https://github.com/evangeloszotos>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function isFunction(fn: any): boolean;
```

Comment:
It's not `export default`: https://github.com/grncdr/js-is-function/blob/master/index.js#L1

Reply:
Oh my, yes, you are right, default export was the wrong approach, I spent now quite some time to figure out how to solve it best. 

if I delete the line with the namespace declaration:
`declare namespace isFunction {}`

then I it is message in visual studio code: 
![image](https://user-images.githubusercontent.com/33234890/51571046-43593b00-1ea1-11e9-9304-e10e6506bcc0.png)

I also tried to autogenerate the .d.ts file with dts-gen and here is what I received: 
![image](https://user-images.githubusercontent.com/33234890/51571119-7e5b6e80-1ea1-11e9-9bf4-d89ccd3d1bae.png)

All my search was bringing me back to that original proposal. 

I am open to a different approach, although I must confess I was able to solve it in a different way so far.

Reply:
You need to use `import isFunction = require("is-function")`. If you still want to use ES module imports, you need to enable `esModuleInterop`.

Reply:
I think know I have it right.

Thank you @DanielRosenwasser and @plantain-00 for your support on this